test/multi.html

```diff
@@ -115,7 +115,7 @@
         assert.equal(deselectEventCounter, 1);
       });
 
-      test('toggle multi from true to false', function() {
+      /* test('toggle multi from true to false', function() {
```

Comment:
is this commented out because it's failing? do we know why it's failing? (i'm sorry!)